static/js/stream_list.js

```diff
@@ -143,8 +143,13 @@ export function build_stream_list(force_rerender) {
     const any_pinned_streams = stream_groups.pinned_streams.length > 0;
     const any_normal_streams = stream_groups.normal_streams.length > 0;
     const any_dormant_streams = stream_groups.dormant_streams.length > 0;
+    const need_section_subheaders =
+        Number.parseInt(any_pinned_streams ? 1 : 0, 10) +
+            Number.parseInt(any_normal_streams ? 1 : 0, 10) +
+            Number.parseInt(any_dormant_streams ? 1 : 0, 10) >=
```

Comment:
These `parseInt`  calls don't make any sense; those values are already integers. I imagine your actual problem was an order-of-operations issue that needed just any parens. I've fixed this by replacing these with parens.